pkg/cmd/server/origin/auth_config.go

```diff
@@ -83,11 +83,9 @@ func BuildAuthConfig(masterConfig *MasterConfig) (*AuthConfig, error) {
 	}
 
 	// Build the list of valid redirect_uri prefixes for a login using the openshift-web-console client to redirect to
-	// TODO: allow configuring this
-	// TODO: remove hard-coding of development UI server
 	assetPublicURLs := []string{}
 	if !options.DisabledFeatures.Has(configapi.FeatureWebConsole) {
-		assetPublicURLs = []string{options.OAuthConfig.AssetPublicURL, "http://localhost:9000", "https://localhost:9000"}
+		assetPublicURLs = []string{options.OAuthConfig.AssetPublicURL}
```

Comment:
I don't think it's that far-fetched for 'oc cluster up' to simply add localhost:9000 as a redirect URI. It's meant to be a tool to help us get environments up for development, and this seems like a common use case. I'll open an issue.

@liggitt can you think of bad things happening because we add that as a default for cluster up?